app/graphql/types/base_argument.rb

```diff
@@ -0,0 +1,4 @@
+module Types
```

Comment:
All of these "base" files were also added in by the graphql gem. I image we might be able to delete them later, but I figured id keep them in while we continue to build in case we might need them.

Reply:
good plan, we can definitely delete them later to increase testing coverages if they are not being used